fiftyone/zoo/datasets/base.py

```diff
@@ -2468,11 +2468,203 @@ def supported_splits(self):
     def supports_partial_downloads(self):
         return True
 
-    def _download_and_prepare(self, dataset_dir, _, split):
+    def _download_and_prepare(self, dataset_dir, version, split):
         num_samples, classes, downloaded = fouo.download_open_images_split(
             dataset_dir,
             split,
+            label_types=self.label_types,
+            classes=self.classes,
+            attrs=self.attrs,
+            image_ids=self.image_ids,
+            num_workers=self.num_workers,
+            shuffle=self.shuffle,
+            seed=self.seed,
+            max_samples=self.max_samples,
             version="v6",
+        )
+
+        dataset_type = fot.OpenImagesDataset()
+
+        if not downloaded:
+            num_samples = None
+
+        return dataset_type, num_samples, classes
+
+
+class OpenImagesV7Dataset(FiftyOneDataset):
+    """Open Images V7 is a dataset of ~9 million images, roughly 2 million of
+    which are annotated and available via this zoo dataset.
+
+    The dataset contains annotations for classification, detection,
+    segmentation, and visual relationship tasks for the 600 boxable object
+    classes.
+
+    This dataset supports partial downloads:
+
+    -   You can specify subsets of data to download via the``label_types``,
+        ``classes``, ``attrs``, and ``max_samples`` parameters
+    -   You can specify specific images to load via the ``image_ids`` parameter
+
+    See :ref:`this page <dataset-zoo-open-images-v6>` for more information
```

Comment:
Unfortunately, when adding new zoo datasets, you must manually add a new entry to the docs here:
https://github.com/voxel51/fiftyone/blob/develop/docs/source/user_guide/dataset_zoo/datasets.rst

It is largely copy-paste from this docstring.

After doing so, you can update this ref to `dataset-zoo-open-images-v7`.